deploy/docker/cp-search-elk/init.sh

```diff
@@ -235,246 +242,11 @@ INDEX="{
-         }
-       },
-       {
-         \"remove\": {
-           \"field\": \"timestamp\",
-           \"ignore_missing\": true,
-           \"ignore_failure\": true
-          }
-       },
-       {
-         \"remove\": {
-           \"field\": \"fields\",
-           \"ignore_missing\": true,
-           \"ignore_failure\": true
-          }
-       },
-       {
-         \"remove\": {
-           \"field\": \"host\",
-           \"ignore_missing\": true,
-           \"ignore_failure\": true
-          }
-       }
-    ]
-}"
-
-curl -H 'Content-Type: application/json' -XPUT localhost:9200/_ingest/pipeline/api_server -d "$API_SRV_PIPELINE"
-
-TINY_PROXY_PIPELINE="{
-    \"description\" : \"Log data extraction pipeline from TinyProxy\",
-    \"processors\": [
-      {
-        \"grok\": {
-          \"field\": \"message\",
-          \"patterns\": [\"%{DATESTAMP:log_timestamp} %{GREEDYDATA} RunOwner: %{DATA:user}, %{GREEDYDATA}\"]
-        }
-      },
-       {
-         \"rename\": {
-           \"field\": \"fields.type\",
-           \"target_field\": \"type\"
-         }
-       },
-       {
-         \"set\": {
-           \"field\": \"service_account\",
-           \"value\": false,
-           \"ignore_failure\": true
-          }
-       },
-        {
-         \"script\": {
-           \"ignore_failure\": false,
-           \"lang\": \"painless\",
-           \"source\": \"ctx.event_id=System.nanoTime()\"
-         }
-       },
-       {
-         \"set\": {
-           \"if\": \"ctx.user.equalsIgnoreCase('$CP_DEFAULT_ADMIN_NAME')\",
-           \"field\": \"service_account\",
-           \"value\": true,
-           \"ignore_failure\": true
-          }
-       },
-       {
-         \"rename\": {
-           \"field\": \"fields.service\",
-           \"target_field\": \"service_name\"
-         }
-       },
-       {
-         \"rename\": {
-           \"field\": \"host.name\",
-           \"target_field\": \"hostname\"
-         }
-       },
-       {
-         \"date\": {
-            \"field\" : \"log_timestamp\",
-            \"target_field\" : \"message_timestamp\",
-            \"formats\" : [\"yy/MM/dd HH:mm:ss\"]
-         }
-       },
-       {
-         \"remove\": {
-           \"field\": \"log_timestamp\",
-           \"ignore_missing\": true,
-           \"ignore_failure\": true
-          }
-       },
-       {
-         \"remove\": {
-           \"field\": \"fields\",
-           \"ignore_missing\": true,
-           \"ignore_failure\": true
-          }
-       },
-       {
-         \"remove\": {
-           \"field\": \"host\",
-           \"ignore_missing\": true,
-           \"ignore_failure\": true
-          }
-       }
-    ]
-}"
-
-curl -H 'Content-Type: application/json' -XPUT localhost:9200/_ingest/pipeline/tiny_proxy -d "$TINY_PROXY_PIPELINE"
+for _pipeline_path in "/etc/search-elk/pipelines/*.json"; do
+  _pipeline_name="$(basename "$_pipeline_path" .json)"
```

Comment:
I think we need mirroring for `"` here, don't you think?

Reply:
Because of the wrapping `$()` the inter quotes are independent from the outer ones. As a result no mirroring is required.

Reply:
ok, sounds good